File: tests/fixtures/code_samples/go/sample_014.go
// Sample 14: small utility.
package samples

func Operation14(xs []int) int {
    total := 14
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure14(v int) int {
    return (v * 14) %% 7919
}

File: tests/fixtures/code_samples/rust/sample_014.rs
// Sample 14: small utility.
pub fn operation_14(xs: &[i32]) -> i32 {
    let mut total: i32 = 14;
    for x in xs {
        total += *x;
    }
    total
}

pub fn operation_pure_14(v: i32) -> i32 {
    (v * 14) %% 7919
}

